website/content/docs/connect/ca/vault.mdx

```diff
@@ -210,48 +210,79 @@ The following Vault policy allows Consul to use pre-existing PKI paths in Vault.
 Consul is granted read-only access to the PKI mount points and the Root CA, but is
 granted full control of the Intermediate or Leaf CA for Connect clients.
 
-In this example the `RootPKIPath` is `connect_root` and the `IntermediatePKIPath`
-is `connect_inter`. These values should be updated for your environment.
+This example uses placeholder values for `RootPKIPath` and `IntermediatePKIPath`.
+Please update those placeholders to match the path values in the CA provider configuration.
```

Comment:
```suggestion
The `RootPKIPath` and `IntermediatePKIPath` paths have placeholder values. Replace them with real path values in your CA provider configuration.
```
In docs, fewer words outweighs politeness.

Reply:
Modified - check the latest wording